ClassHierarchy_hw/Program.cs

```diff
@@ -0,0 +1,204 @@
+            CreateIngredients(ref ingredients, new Ingredient("Pinaple", 50.0, 100, "gramm", "fruit"));
+            CreateIngredients(ref ingredients, new Ingredient("Sour Cherry", 77.0, 100, "gramm", "fruit"));
+            CreateIngredients(ref ingredients, new Ingredient("Brown Rice", 111.0, 100, "gramm", "grains"));
+            CreateIngredients(ref ingredients, new Ingredient("Corn", 90.0, 100, "gramm", "grains"));
+            CreateIngredients(ref ingredients, new Ingredient("Chiken", 192.0, 100, "gramm", "protein"));
+            CreateIngredients(ref ingredients, new Ingredient("Fish (salmon)", 252.0, 100, "gramm", "protein"));
+            CreateIngredients(ref ingredients, new Ingredient("Eggs", 155.0, 100, "gramm", "protein"));
+            CreateIngredients(ref ingredients, new Ingredient("Shrimp", 99.0, 100, "gramm", "protein"));
+            CreateIngredients(ref ingredients, new Ingredient("Salsa", 36.0, 100, "gramm", "dressing"));
+            CreateIngredients(ref ingredients, new Ingredient("Chutney", 25.0, 100, "gramm", "dressing"));
+            CreateIngredients(ref ingredients, new Ingredient("Guacamole", 45.0, 100, "gramm", "dressing"));
+            CreateIngredients(ref ingredients, new Ingredient("Macadamia Nuts", 718.0, 100, "gramm", "yum-ify!"));
+            CreateIngredients(ref ingredients, new Ingredient("Goat Cheese", 364.0, 100, "gramm", "yum-ify!"));
+            CreateIngredients(ref ingredients, new Ingredient("Cashews", 553.0, 100, "gramm", "yum-ify!"));
+            CreateIngredients(ref ingredients, new Ingredient("Croutons", 407.0, 100, "gramm", "yum-ify!"));
+
+            // Calling the interface
+            SaladBuilderInterface(ingredientTypes, ingredients);
+        }
+
+        // A method that creates categories
+        public static void CreateIngredientTypes(ref IngredientType[] ingredientTypes, IngredientType ingredientType)
+        {
+            ArrayPush(ref ingredientTypes, ingredientType);
+        }
+
+        // A method that creates ingredients
+        public static void CreateIngredients(ref Ingredient[] ingredientTypes, Ingredient ingredientType)
+        {
+            ArrayPush(ref ingredientTypes, ingredientType);
+        }
+
+        // Interface method to ask a person about what ingredients to add and returns a salad
+        public static void SaladBuilderInterface(IngredientType[] ingredientTypes, Ingredient[] ingredients)
+        {
+            int optionCounter = 0;
+            Console.WriteLine(" Hi this is a SALAD BUILDER NIKITA 3000");
+            Console.WriteLine("///////////");
+            Console.WriteLine(" Let's build our own salad!");
+            Console.WriteLine("///////////");
+            Console.WriteLine(" Please choose your own content of the salad");
+            Console.WriteLine(" You can choose from: ");
+            Console.WriteLine("//////////");
```

Comment:
Please, dont use such an output, just make every new statement in a separate line in console